src/exceptions/Bootloader-exception

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="Bootloader-exception" name="Bootloader Distribution Exception">
+      <crossRefs>
+         <crossRef>https://github.com/pyinstaller/pyinstaller/blob/develop/COPYING.txt</crossRef>
+      </crossRefs>
+  
+      <titleText>Bootloader Exception</titleText>
+<optional>--------------------</optional>
+<p>In addition to the permissions in the GNU General Public License, the
+authors give you unlimited permission to link or embed compiled bootloader
+and related files into combinations with other programs, and to distribute
+those combinations without any restriction coming from the use of those
+files. (The General Public License restrictions do apply in other respects;
+for example, they cover modification of the files, and distribution when
+not linked into a combine executable.)</p>
```

Comment:
[Upstream][1] and [the GCC exception 2.0][2] both have this typo too, but I'm pretty sure “combine executable” should be “combined executable” (compare “[combined work][3]”).  If folks agree, I think we want to mark that up with:

```
not linked into a <alt name="combined" match="combined|combine">combined</alt> executable.)</p>
```

[1]: https://github.com/pyinstaller/pyinstaller/blob/v3.3/COPYING.txt#L25
[2]: https://github.com/spdx/license-list-XML/blob/master/src/exceptions/GCC-exception-2.0.xml#L18
[3]: https://github.com/spdx/license-list-XML/blob/master/src/GPL-3.0.xml#L498

Reply:
we usually don't "fix" typos especially when they appear consistently, but in the chance that someone else has fixed it downstream, I'm fine with adding this. (can you do so?)

Reply:
that's fine, please add :)

Reply:
> … I'm fine with adding this. (can you do so?)

I don't have push access to this repo, so I can't update your branch directly.  Instead, I've filed #515 against your branch; merging #515 will update this PR.